src/exporter/exporter.go

```diff
@@ -137,11 +137,7 @@ func (e *Exporter) Kill() {
 		return
 	default:
 		e.cancel()
-		if err := e.cmd.Wait(); err != nil {
```

Comment:
This shouln't be super useful, but are you sure that the wait is called before?
When the wait is called e.Done channel is closed we do not wait again, we simply return

```
	go func() {
		e.cmd.Wait()
		log.Debug("exporter has stopped")
		close(e.Done)
	}()
```